$MirOS: ports/net/djbdns/patches/patch-src_scan_h,v 1.2 2005/12/27 17:46:18 tg Exp $
--- src/scan.h.orig	Sun Feb 11 21:11:23 2001
+++ src/scan.h	Sat Mar 26 16:17:11 2016
@@ -1,6 +1,8 @@
 #ifndef SCAN_H
 #define SCAN_H
 
+#include <stdint.h>
+
 extern unsigned int scan_uint(const char *,unsigned int *);
 extern unsigned int scan_xint(const char *,unsigned int *);
 extern unsigned int scan_nbbint(const char *,unsigned int,unsigned int,unsigned int,unsigned int *);
@@ -8,6 +10,7 @@ extern unsigned int scan_ushort(const ch
 extern unsigned int scan_xshort(const char *,unsigned short *);
 extern unsigned int scan_nbbshort(const char *,unsigned int,unsigned int,unsigned int,unsigned short *);
 extern unsigned int scan_ulong(const char *,unsigned long *);
+extern unsigned int scan_uint32(const char *,uint32_t *);
 extern unsigned int scan_xlong(const char *,unsigned long *);
 extern unsigned int scan_nbblong(const char *,unsigned int,unsigned int,unsigned int,unsigned long *);
 
